### PR TITLE
x11vnc: new package at 0.9.16

### PR DESCRIPTION
**Pull request by mcpcpc**
Please fill out the template.

Read: https://k1ss.org/wiki/kiss/style-guide

---

## Description of package

x11vnc allows one to view remotely and interact with real X displays (i.e. a display corresponding to a physical monitor, keyboard, and mouse) with any VNC viewer.

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```

/var/db/kiss/installed/x11vnc/version
/var/db/kiss/installed/x11vnc/sources
/var/db/kiss/installed/x11vnc/patches/libressl.patch
/var/db/kiss/installed/x11vnc/patches/bugfixes.patch
/var/db/kiss/installed/x11vnc/patches/
/var/db/kiss/installed/x11vnc/manifest
/var/db/kiss/installed/x11vnc/depends
/var/db/kiss/installed/x11vnc/checksums
/var/db/kiss/installed/x11vnc/build
/var/db/kiss/installed/x11vnc/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/x11vnc.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/applications/x11vnc.desktop
/usr/share/applications/
/usr/share/
/usr/bin/x11vnc
/usr/bin/Xdummy
/usr/bin/
/usr/

```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/x11vnc/build [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+patch -p1 < bugfixes.patch
+patch -p1 < libressl.patch
```
Comment (dylanaraps) [inline community/x11vnc/build:4]:
```sh
for patch in *.patch; do
    patch -p1 < "$patch"
done

File: community/x11vnc/build [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+./autogen.sh
```
Comment (dylanaraps) [inline community/x11vnc/build:6]: No way to avoid this (no release tarballs lying around anywhere?)

Comment (mcpcpc) [reply community/x11vnc/build:6]: Found a more *recent* tarball from fossies (https://fossies.org/linux/privat/x11vnc-0.9.16.tar.gz/).  Patching still required though.  

Updated bugfixes.patch and source files per latest commit.  Not change to manifest.

Comment (mcpcpc) [reply community/x11vnc/build:6]: actually... its the same tarball -.- scratch my previous statement.  still the most recent that i could find.

**Pull request merged**